.gitignore

```diff
@@ -8,3 +8,4 @@ node_modules
 /.direnv/
 .next
 .vercel
+tsconfig.tsbuildinfo
```

Comment:
https://github.com/vercel/next.js/discussions/30815

Reply:
Checked the preview on my end, and archived shows are being served from a secure s3 url. Approved!

Reply:
Thanks!